src/integrationtest/get_pytest_tmpdir.py

```diff
@@ -0,0 +1,24 @@
+# 30-Dec-2025, KAB: this function determines the root of the current pytest output
+# directory (i.e. the current pytest "tmpdir"). It does this by checking the relevant
+# environmental variables to see if any of them have been set.
+# The order in which the env vars are checked is based on the precedence that the
+# pytest framework uses.
```

Comment:
Maybe include the link?
`https://docs.pytest.org/en/stable/how-to/tmp_path.html#temporary-directory-location-and-retention`